todo_app/trello_api.py

```diff
@@ -1,7 +1,7 @@
 from flask import session, current_app as app
```

Comment:
Not sure why we have so many code changes in this review?

Reply:
I cleaned up the project structure and moved all the code into it's own folder away from the config type stuff and docker files... this meant I had to change some class paths as it broke some of the import statements.